worldwind/src/main/java/gov/nasa/worldwind/util/Tile.java

```diff
@@ -425,25 +432,21 @@ protected BoundingBox getExtent(RenderContext rc) {
      * @return the distance in meters
      */
     protected double distanceToCamera(RenderContext rc) {
-        if (this.sector.contains(rc.camera.latitude, rc.camera.longitude)) {
-            return rc.camera.altitude;
-        }
-
-        if (this.samplePoints == null) {
-            this.samplePoints = rc.globe.geographicToCartesianGrid(this.sector, 3, 3, null, 1.0f, null, new float[27], 0, 0);
+        // determine the nearest latitude
+        double nearestLat = WWMath.clamp(rc.camera.latitude, this.sector.minLatitude(), this.sector.maxLatitude());
+        // determine the nearest longitude and account for the antimeridian discontinuity
+        double nearestLon;
+        double lonDifference = rc.camera.longitude - this.sector.centroidLongitude();
+        if (lonDifference < -180.0) {
+            nearestLon = WWMath.clamp(rc.camera.longitude + 360.0, this.sector.minLongitude(), this.sector.maxLongitude());
+        } else if (lonDifference > 180.0) {
+            nearestLon = WWMath.clamp(rc.camera.longitude - 360.0, this.sector.minLongitude(), this.sector.maxLongitude());
+        } else {
+            nearestLon = WWMath.clamp(rc.camera.longitude, this.sector.minLongitude(), this.sector.maxLongitude());
         }
 
-        double minDistanceSq = Double.MAX_VALUE;
-        for (int i = 0, len = this.samplePoints.length; i < len; i += 3) {
-            double dx = rc.cameraPoint.x - this.samplePoints[i];
-            double dy = rc.cameraPoint.y - this.samplePoints[i + 1];
-            double dz = rc.cameraPoint.z - this.samplePoints[i + 2];
-            double distanceSq = (dx * dx) + (dy * dy) + (dz * dz);
-            if (minDistanceSq > distanceSq) {
-                minDistanceSq = distanceSq;
-            }
-        }
+        rc.geographicToCartesian(nearestLat, nearestLon, this.heightLimits[0] * rc.verticalExaggeration, WorldWind.ABSOLUTE, this.nearestPoint);
```

Comment:
I think this code would be more self-documenting if we created a local variable that captures altitude:
```java
float minHeight = (float) (this.heightLimits[0] * verticalExaggeration);
```
What do you think?